Create ui-c8y-1023-4-1-update-web-sdk-to-v-1023-guide.md (#3946)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-4-1-update-web-sdk-to-v-1023-guide.md

---
date: ""

> title: Update Web SDK to v1023 guide

Reviewer: ```suggestion
title: Web SDK update to v1023 guide added
```

> title: Update Web SDK to v1023 guide

Reviewer: it's not 'version 1023 of guide' but 'guide about how to update web sdk to v1023'

> title: Web SDK update to v1023 guide added

Reviewer 2: ```suggestion
title: Information on migrating to the Web SDK v1023 added to the Developer Codex
```

> title: Information on migrating to the Web SDK v1023 added to the Developer Codex

Reviewer 2: ```suggestion
title: Information on migrating to Web SDK v1023 added to the Developer Codex
```

product_area: Application enablement & solutions
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64954
version: 1023.4.1
---

> Co-authored-by: Jakub Drewniak <103751383+jdre-c8y@users.noreply.github.com>

Reviewer: ```suggestion
The Web SDK update guide has been enriched with new entry to cover the latest changes and improvements in version 1023 of the Web SDK. This guide now provides more detailed information on how to use the new features and APIs introduced in this version, such as global time context. With this update, developers can leverage the full potential of the Web SDK to build powerful and efficient web applications.
```

> The Web SDK update guide has been enriched with new entry to cover the latest changes and improvements in version 1023 of the Web SDK. This guide now provides more detailed information on how to use the new features and APIs introduced in this version, such as global time context. With this update, developers can leverage the full potential of the Web SDK to build powerful and efficient web applications.

Reviewer 2: Does this refer to a description of how to update to version 1023 or what you can do with version 1023? Can we add a link to the text even if it is not yet live?

Reviewer: how to update to 1023 and what are the most important changes coming with 1023 version of web sdk. sure, we can add a link, it's available now.

> The Web SDK update guide has been enriched with new entry to cover the latest changes and improvements in version 1023 of the Web SDK. This guide now provides more detailed information on how to use the new features and APIs introduced in this version, such as global time context. With this update, developers can leverage the full potential of the Web SDK to build powerful and efficient web applications. Check [the guide here](https://cumulocity.com/codex/migration-guides/updating-web-sdk-version/overview#version-specific-migration).

Reviewer 2: ```suggestion
The Web SDK update documentation has been enhanced with additional information to cover the latest changes and improvements in version 1023 of the Web SDK. The documentation now provides detailed information on how to use the new features and APIs introduced in this version, such as the global time context. See [Updating the Web SDK version](https://cumulocity.com/codex/migration-guides/updating-web-sdk-version/overview#version-specific-migration) in the Developer Codex.
```
